docs/prepping-local-system-runbook-windows.md

```diff
@@ -246,13 +246,16 @@ Pegasystems uses a standard naming practice of hostname/product/image:tag. Pega
 `platform/installer`   | A utility image with which you install all of the Pega-specific rules and database tables in the “Pega” database that you have configured for your deployment. This installation is required before a deployment can take place.| `<version>` |
 `platform/pega`  | (Download required) Deploys Pega Platform with its customized version of the Tomcat application server.| `<version>` or `<version>-YYYYMMDD` |
 `platform/search` | (Download required) Deploys the required search engine for Pega Platform search and reporting capabilities. This Docker image contains Elasticsearch and includes all required plugins.| `<version>` or `<version>-YYYYMMDD` |
-
+`platform-services/search-n-reporting-service`  | [optional and compatible only with Pega version 8.6 and later] (Download required) An image that deploys the required search engine for the latest generation of search and reporting capabilities in Pega Infinity as a backing service. This Docker image contains Pega Search and Reporting Services. For more information about this service, see [Instructions to configure the Pega backingservices](charts/backingservices/README.md). | `<version>` |
```

Comment:
[optional and compatible only with Pega version 8.6 and later] (Download required)->>edit>>Deploys the latest Pega generation of search and reporting capabilities in Pega Infinity on dedicated, backing service nodes. This Docker image contains Pega Search and Reporting Services and allows you to optimize Elasticsearch processing in your Pega deployment. To optimize Elasticsearch processing, you must  deploy this image and the matching release of the platform/search image. <<edit<<-For more information about this service, see [Instructions to configure the Pega backingservices](charts/backingservices/README.md). | `<version>`